build.gradle

```diff
@@ -202,7 +217,7 @@ dependencies {
     implementation "com.typesafe.scala-logging:scala-logging_${scalaBinaryVersion}:3.9.0"
     implementation "org.slf4j:log4j-over-slf4j:${slf4jVersion}"
 
-    implementation(group: 'com.github.LBNL-UCB-STI', name: 'r5', version: 'v4.5.3_linkRadiusMeters') {
+    implementation(group: 'com.github.LBNL-UCB-STI', name: 'r5', version: 'v4.8.1_linkRadiusMeters') {
```

Comment:
This is based on changes from [beam-patches-4.8](https://github.com/LBNL-UCB-STI/r5/tree/beam-patches-4.8)